enn_zoo/pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
+name = "enn_zoo"
+version = "0.1.0"
+description = "A collections of environments which can be used with entity-neural-networks"
+authors = ["Clemens Winter <clemenswinter1@gmail.com>"]
+
+[tool.poetry.dev-dependencies]
+griddly = {version = "^1.2.15", optional=true}
+entity_gym = {path = "../entity_gym", develop = true}
+rogue_net = {path = "../rogue_net", develop = true}
```

Comment:
This dependency seems to be unused? I don't see a fundamental reason why entity_zoo should be tied to a specific network architecture/implementation.

Reply:
I think this was just me trying to get poetry to be less of a pain. Wasn't entirely sure what I was doing here to be honest